devel/gettext/Portfile

```diff
@@ -6,119 +6,193 @@ PortGroup               muniversal 1.0
 PortGroup               clang_dependency 1.0
 
 name                    gettext
-epoch                   2
-version                 0.19.8.1
-revision                2
+version                 0.20.2
 categories              devel
 maintainers             {ryandesign @ryandesign}
-# libs are LGPL and executables are GPL
-license                 {LGPL-2.1+ GPL-3+}
 homepage                https://www.gnu.org/software/gettext/
 master_sites            gnu
 platforms               darwin freebsd linux
 use_parallel_build      yes
 
-description             GNU internationalization (i18n) and localization (l10n) library
-
-long_description        ${name} is the ${description}, used for writing multilingual programs.
-
-checksums               rmd160  29a5f29d186fe2aece5c450fd1a743b819c02c5b \
-                        sha256  ff942af0e438ced4a8b0ea4b0b6e0d6d657157c5e2364de57baa279c1c125c43
-
-# https://trac.macports.org/ticket/58323
-patchfiles              write-java.c.patch
+checksums               rmd160  32e99c55361970315dd9d43f9e36dd43f05fc09e \
+                        sha256  ecb9d0908ca41d5ca5fef974323b3bba6bec19eebba0b44f396de98cfcc089f1 \
+                        size    23717789
 
 # https://trac.macports.org/ticket/31167
 compiler.blacklist-append   {clang < 211.10.1}
 
-set localbindir         ${workpath}/bin
+configure.cppflags      -no-cpp-precomp
```

Comment:
This option has done nothing since gcc 3.3 and TBH should have been removed years ago.

Reply:
I was looking suspiciously at that line and couldn't remember if it was still needed on older systems. I've refreshed my memory now and agree with you, it should go. I should just take some time to remove it from all ports.